Runtime/Scripts/AudioSource.cs

```diff
@@ -28,10 +34,16 @@ public RtcAudioSource(AudioSource source)
             var request = new FfiRequest();
             request.NewAudioSource = newAudioSource;
 
-            var resp = FfiClient.SendRequest(request);
+            Init(request, source);
+        }
+
+        async void Init(FfiRequest request, AudioSource source)
```

Comment:
Considering `Init` has become an `async` method you can't call it from `ctor` and throw it into the void.
Thus, it's just spinning without any control.
Consider introducing an async factory method that will return a task. In this case it will be possible to create `RtcAudioSource` from outside and await its completion.